lib/jekyll-docs.rb

```diff
@@ -20,17 +18,13 @@ def init_with_program(prog)
       end
 
       def process(opts)
-        Dir.mktmpdir do |dest_dir|
-          options = opts.merge({
-            "serving"     => true,
-            "watch"       => false,
-            "config"      => File.expand_path("../../site/_config.yml", __FILE__),
-            "source"      => File.expand_path("../../site", __FILE__),
-            "destination" => dest_dir
-          })
-          Jekyll::Commands::Build.process(options)
-          Jekyll::Commands::Serve.process(options)
-        end
+        options = opts.merge({
+          "serving"     => true,
+          "watch"       => false,
+          "destination" => File.expand_path("../../site", __FILE__),
```

Comment:
I think you still need `"source"` in here! `"destination"` is the output directory.

The only thing you can change about this method is: 

- remove `Jekyll::Commands::Build.process(options)`
- remove `"config"` from `opts`.

The `Dir.mktmpdir`, etc must all remain.